Blog/sammoss.md

```diff
@@ -0,0 +1,8 @@
+#Jan 26
+
+The Public Humanities 
+
+The very personal nature of Risam's article resonated with me, because while not a person of color, I can empathize with some, if not all, of the barriers to coding. I appreciate her framing the discussion in the context of the Torn Apart/Separados projects, because she provides a window into the experience of just jumping in and trying to figure 'it' out. Actually, she resonates hard because she describes some of my own thoughts ("How do I get the code into a thingy"? for one among many. Also, did she really need to use it as an excuse to watch Marvel "yes, including Thor".) She made me laugh and be inspired. Seriously, though, I return to the quandary she presents - the double and triple binds of computational ethic studies. Critically, she made me aware of these issues in the field, so that I can move forward in this space with more awareness of the barriers faced by my peers and help institute change in the moments in which that's within my power. 
```

Comment:
I also loved the personal nature that was contextualized by Ethnic Studies as a field, and the added bind of that "computational". What is hard about teaching computational skills is that if those learning don't have a meaningful project to work on it is hard to get things to stick, and Risam discussed these attempts at learning prior to having a true application